src/preferences/dialog/dlgprefmixer.cpp

```diff
@@ -220,245 +249,182 @@ void DlgPrefMixer::slotNumDecksChanged(double numDecks) {
-        box->addItem(kNoEffectString);
-        int i = 1;
+        // Get currently loaded EQ effect
+        auto pChainSlot = m_pEffectsManager->getEqualizerEffectChain(
+                PlayerManager::groupForDeck(deck));
+        DEBUG_ASSERT(pChainSlot);
+        auto pEffectSlot = pChainSlot->getEffectSlot(0);
+        DEBUG_ASSERT(pEffectSlot);
+        const EffectManifestPointer pLoadedManifest =
+                pEffectSlot->getManifest();
+
+        pBox->clear();
+        // Add empty item at the top (no effect)
+        pBox->addItem(kNoEffectString);
+        int currentIndex = 0; // store it as default selection
         for (const auto& pManifest : pManifestList) {
-            box->addItem(pManifest->name(), QVariant(pManifest->uniqueId()));
-            if (pCurrentlySelectedManifest &&
-                    pCurrentlySelectedManifest.data() == pManifest.data()) {
+            pBox->addItem(pManifest->name(), QVariant(pManifest->uniqueId()));
+            int i = pBox->count() - 1;
+            // <b> makes the effect name bold. Also, like <span> it serves as hack
+            // to get Qt to treat the string as rich text so it automatically wraps long lines.
+            pBox->setItemData(i,
+                    QVariant(QStringLiteral("<b>%1</b><br/>%2")
+                                     .arg(pManifest->name(),
+                                             pManifest->description())),
+                    Qt::ToolTipRole);
+            if (pLoadedManifest &&
+                    pLoadedManifest.data() == pManifest.data()) {
                 currentIndex = i;
             }
-            ++i;
         }
-        if (pCurrentlySelectedManifest == nullptr) {
-            // Configured effect has no id, clear selection
-            currentIndex = 0;
-        } else if (currentIndex < 0) {
-            // current selection is not part of the new list
-            // So we need to add it
-            box->addItem(pCurrentlySelectedManifest->shortName(),
-                    QVariant(pCurrentlySelectedManifest->uniqueId()));
-            currentIndex = i + 1;
+        if (pLoadedManifest && currentIndex == 0) {
+            // Current selection is not part of the new list so we need to add it
+            pBox->addItem(pLoadedManifest->displayName(),
+                    QVariant(pLoadedManifest->uniqueId()));
+            currentIndex = pBox->count() - 1;
+            pBox->setItemData(currentIndex,
+                    QVariant(QStringLiteral("<b>%1</b><br/>%2")
+                                     .arg(pLoadedManifest->name(),
+                                             pLoadedManifest->description())),
+                    Qt::ToolTipRole);
+            // Deactivate item to hopefully clarify the item is not an EQ
+            const QStandardItemModel* pModel =
+                    qobject_cast<QStandardItemModel*>(pBox->model());
+            DEBUG_ASSERT(pModel);
+            auto pItem = pModel->item(currentIndex);
```

Comment:
clang-tidy. also affects `slotSingleEqToggled`
```suggestion
            auto* pItem = pModel->item(currentIndex);
```